frame/babe/src/mock.rs

```diff
@@ -70,19 +81,40 @@ impl frame_system::Trait for Test {
 
 impl_opaque_keys! {
 	pub struct MockSessionKeys {
-		pub dummy: UintAuthorityId,
+		pub babe_authority: super::Module<Test>,
 	}
 }
 
 impl pallet_session::Trait for Test {
 	type Event = ();
 	type ValidatorId = <Self as frame_system::Trait>::AccountId;
 	type ShouldEndSession = Babe;
-	type SessionHandler = (Babe,Babe,);
-	type SessionManager = ();
+	type SessionHandler = (Babe,);
```

Comment:
[changed upstream](https://github.com/paritytech/substrate/commit/18334ee173f0ec4b62d2ca05c5b0c1f0b05b1b17#diff-77d405d7362bce705009a3f7e5bf95d0R116)
Is there a reason not to change to that?
```suggestion
	type SessionHandler = <MockSessionKeys as OpaqueKeys>::KeyTypeIdProviders;
```